### ISSUE TITLE
'load_sse_c' option not working

### ISSUE THREAD

**Issue by soufrk** [s3fs-fuse/s3fs-fuse#388]
I have been working with SSE-C capabilities of 's3fs', and after reading through, what I found out is a custom key can be specified at the time of mounting. What if individual files in a single bucket are encrypted with different keys. Is there a specific way to mount a bucket simply, and make download/fetch requests for a particular file with a particular key ?

Let me know, in case the problem statement is not clear.

**EDIT**
Additionally, just now I discovered that reading encrypted file is not working even after mounting with custom key.

Hoping so that the following command is correct for SSE-C

```sudo s3fs my-own-bucket /var/s3 -o allow_other -o use_sse=custom:/etc/mykey```

I would also like to add the fact the file upload(encrypt)/download(decrypt) is working fine through SDK.

**EDIT**
I realized that there is a 'load_sse_c' option. But I am not able to get it working. As, the syntax for it's seems to be unclear.
Tried ```-o load_sse_c=/etc/mykey``` and ```-o load_sse_c /etc/mykey``` , yet suffers from syntactical errors.

**Comment by soufrk** [comment]
Closing this one as posting the question directly to enhancement thread.

**Comment by soufrk** [comment]
Reopening, as unable to get issue raised in enhancement ticket.
Unable to get 'load_sse_c' option working. Results of trials below,
```
$ sudo s3fs mybucket /var/s3 -o allow_other -o load_sse_c=/etc/mykey
fuse: unknown option `load_sse_c=/etc/mykey'
$ sudo s3fs mybucket /var/s3 -o allow_other -o use_sse=custom -o load_sse_c /etc/mykey
s3fs: specified unknown third optioni(/etc/mykey).
$ sudo s3fs mybucket /var/s3 -o allow_other -o use_sse=custom:/etc/mykey -o load_sse_c
fuse: unknown option `load_sse_c'
$ sudo s3fs mybucket /var/s3 -o allow_other -o use_sse=custom -o load_sse_c
s3fs: something wrong about SSE options.
$ sudo s3fs mybucket /var/s3 -o allow_other -o use_sse=custom -o load_sse_c /etc/mykey
s3fs: specified unknown third optioni(/etc/mykey).
$ sudo s3fs mybucket /var/s3 -o allow_other -o use_sse=custom -o load_sse_c=/etc/mykey
fuse: unknown option `load_sse_c=/etc/mykey'
$ sudo s3fs mybucket /var/s3 -o allow_other -o use_sse=c -o load_sse_c=/etc/mykey
fuse: unknown option `load_sse_c=/etc/mykey'
$ sudo s3fs mybucket /var/s3 -o allow_other -o use_sse=c -o load_sse_c /etc/mykey
```

Any help will be appreciated.

**Comment by ggtakec** [comment]
@soufrk I'm very sorry for my late reply.
I check your pointed out problems, it has been fixed by #409 
I closed this issue, but if you found a bug about this, please reopen.

Regards,
